web/src/js/ducks/ui/flow.js

```diff
@@ -1,18 +1,22 @@
 import * as flowsActions from '../flows'
+import { getDiff } from "../../utils"
+
 import _ from 'lodash'
 
 export const SET_CONTENT_VIEW = 'UI_FLOWVIEW_SET_CONTENT_VIEW',
              DISPLAY_LARGE    = 'UI_FLOWVIEW_DISPLAY_LARGE',
              SET_TAB          = "UI_FLOWVIEW_SET_TAB",
              START_EDIT       = 'UI_FLOWVIEW_START_EDIT',
              UPDATE_EDIT      = 'UI_FLOWVIEW_UPDATE_EDIT',
-             STOP_EDIT        = 'UI_FLOWVIEW_STOP_EDIT'
+             STOP_EDIT        = 'UI_FLOWVIEW_STOP_EDIT',
+             UPLOAD_CONTENT   = 'UI_FLOWVIEW_UPLOAD_CONTENT'
 
 
 const defaultState = {
     displayLarge: false,
     modifiedFlow: false,
     contentView: 'ViewAuto',
+    lastFileUpload: false,
```

Comment:
Generally speaking, after receiving an update for the currently displayed flow, the contentloader should rerender. Most extreme example: If I upload a new flow content in window a, it should rerender the flow in window b. if that's not the case, we should fix this issue and not add band-aids.